planning/behavior_velocity_intersection_module/src/util.cpp

```diff
@@ -778,15 +778,15 @@ std::vector<DescritizedLane> generateDetectionLaneDivisions(
     auto & branch = branches[(ind2id[src])];
     int node_iter = ind2id[src];
     while (true) {
-      const auto & dsts = adjacency[(id2ind[node_iter])];
+      const auto & distances = adjacency[(id2ind[node_iter])];
```

Comment:
@HansRobo I was meaning to abbreviate `destination` by `dst` in contrast to `source` and `src`. Can you substitute `destinations` instead ? Thanks

Reply:
Thanks! I'll fix it

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/4716/commits/c059813b14702c5bc82300a4e31a96027443a525